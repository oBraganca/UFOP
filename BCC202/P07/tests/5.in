150 57 145 47 113 129 122 140 87 67 86 89 149 23 94 6 41 30 43 121 49 24 115 120 73 16 17 58 64 107 66 79 54 62 50 101 83 68 132 93 139 13 72 45 15 44 85 56 69 2 124 61 111 53 91 21 100 146 88 36 81 117 4 116 128 51 104 29 8 28 127 108 96 150 18 137 133 25 77 31 138 103 131 134 55 12 141 34 105 125 60 123 130 48 118 71 27 90 80 37 33 74 112 32 95 20 52 75 26 144 126 59 102 135 142 109 40 119 19 35 5 10 65 98 148 11 110 106 14 39 84 99 3 42 38 7 97 70 1 63 143 78 22 147 76 114 82 136 46 9 92
178 151 28 177 23 35 4 59 178 176 104 67 26 46 156 72 21 157 101 169 147 7 50 137 19 8 52 165 100 174 87 58 9 17 83 168 61 171 114 47 45 44 56 74 31 105 95 89 2 170 84 13 78 16 116 53 6 115 54 127 82 63 135 138 27 43 154 142 139 143 118 117 167 130 173 11 79 159 18 91 162 93 128 51 144 29 106 32 24 22 155 158 33 134 92 64 126 1 85 166 103 94 121 86 41 122 81 55 161 140 141 62 123 119 42 110 149 15 80 133 102 124 160 48 153 30 70 20 152 65 5 109 37 36 132 172 69 129 131 146 71 150 96 77 97 3 108 120 68 14 25 60 107 66 88 76 39 38 145 98 148 112 163 125 90 10 164 136 73 49 113 40 111 175 75 34 57 12 99
76 60 12 44 38 51 2 76 3 45 8 49 37 70 74 15 39 72 14 28 50 29 11 73 10 21 54 43 41 6 34 32 9 18 17 31 52 48 58 71 4 66 40 20 47 67 63 1 61 55 7 65 46 23 16 64 36 27 25 57 53 42 33 69 19 75 62 24 26 35 59 5 13 56 68 22 30
21 12 5 10 4 6 18 14 21 7 16 9 19 15 3 20 1 17 13 2 11 8
123 48 12 37 74 63 67 114 57 23 118 5 22 100 38 112 13 9 39 101 115 79 71 70 21 52 58 29 97 30 31 83 36 99 8 122 82 113 7 108 72 2 43 69 77 81 66 105 90 80 10 92 50 55 106 98 14 34 6 49 59 73 95 11 116 104 32 19 93 86 120 41 107 121 17 16 88 111 123 102 15 45 62 85 56 76 26 84 1 18 119 54 110 20 87 89 42 64 35 28 94 33 46 78 103 4 24 51 47 60 109 40 44 27 3 117 91 96 53 25 68 75 65 61
85 53 76 42 14 23 37 48 81 64 25 50 61 35 70 26 19 10 13 62 67 11 59 55 41 27 38 1 18 84 2 63 78 57 46 66 31 79 83 4 51 29 75 20 28 7 43 39 5 12 16 32 56 45 6 3 52 40 85 22 69 21 34 80 49 44 65 36 73 77 58 30 15 8 24 82 68 47 71 33 17 60 54 72 9 74
198 95 173 160 115 64 101 157 19 61 124 111 59 29 149 48 21 45 150 175 152 54 52 184 112 103 2 182 1 125 192 67 90 5 85 191 131 196 186 88 128 127 7 68 133 120 27 12 70 188 161 99 50 163 154 66 108 139 104 35 16 25 172 37 76 46 36 121 118 123 55 71 39 181 82 194 94 72 23 164 176 113 179 30 86 62 143 13 58 156 24 155 57 20 79 100 32 63 47 107 26 146 22 109 138 89 77 147 119 187 87 8 91 142 174 148 51 96 171 136 195 178 92 144 198 43 126 83 158 117 41 78 81 44 97 80 122 151 102 93 135 18 145 17 74 183 49 141 166 140 98 116 159 129 114 42 33 132 168 162 73 14 106 10 3 31 4 177 11 38 105 197 9 60 53 167 15 6 40 153 134 69 165 28 130 137 169 185 84 170 65 193 110 180 56 75 34 190 189
33 8 17 27 10 29 5 33 21 4 24 28 19 23 26 15 9 6 30 22 12 7 18 2 31 20 32 14 3 11 13 25 16 1
167 57 103 99 114 50 87 75 115 125 37 69 131 111 152 30 96 164 110 53 55 167 151 23 122 15 62 146 130 90 6 147 133 116 149 22 5 31 105 8 94 93 35 65 92 166 76 79 47 157 1 38 106 142 117 84 17 154 148 95 118 67 77 119 66 113 19 132 18 134 7 58 48 60 165 3 40 141 89 13 98 163 88 24 123 2 36 52 129 16 27 97 78 161 100 108 128 26 54 45 21 73 83 49 155 25 135 138 124 11 64 107 127 44 14 43 4 143 156 46 162 144 81 28 29 101 70 91 34 158 153 140 85 12 39 102 104 32 126 41 160 63 136 10 42 159 68 121 80 33 137 9 51 120 150 139 112 86 20 109 74 59 82 72 145 71 56 61
131 5 19 78 49 75 126 83 107 98 118 116 43 6 23 38 125 80 7 91 102 103 56 14 115 113 40 90 65 94 99 13 9 88 117 84 120 30 109 101 85 41 48 11 54 71 119 66 122 108 17 111 121 16 52 86 47 45 4 110 46 35 112 95 67 62 57 27 77 22 124 25 128 24 12 81 8 114 127 59 64 18 82 129 72 51 105 10 104 32 2 131 26 60 44 36 58 97 63 3 96 130 42 92 33 20 76 50 70 31 74 87 106 69 53 55 68 123 39 37 15 61 34 93 100 1 21 73 28 29 79 89
0
