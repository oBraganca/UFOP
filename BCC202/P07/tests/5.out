Carlos
Carlos
Marcelo
Marcelo
Carlos
Carlos
Carlos
Carlos
Carlos
Carlos
